client/src/components/campaignCreatorComponents/RadioButtonListCC.tsx

```diff
@@ -0,0 +1,39 @@
+import { Checkbox, Flex, Heading } from '@radix-ui/themes';
+import { TLCheckbox, TLRadioGroupItem } from '../primitiveComponents/Primitives';
+
+interface OptionListCCProps {
+    header: string;
+    options: string[];
+    data: Record<string, boolean>;
+    setData: React.Dispatch<React.SetStateAction<{}>>;
+}
+
+const RadioButtonListCC = ({ header, options, data, setData }: OptionListCCProps) => {
```

Comment:
Should rename this interface to match function name

Reply:
fixed